### PR TITLE
Added a few details to README.md 

### PR DESCRIPTION
**Pull request by MRbiomedical**
Added some details on getting a api token, integrating the bot into a server, and where config.json comes into the process

Also put in a reminder that `dotnet run` must be run in the DiscordBot directory as opposed to the project root

### CODE REVIEW

**Comment by MRbiomedical** [comment]
don't mind the commit names

**Comment by joefroh** [comment]
First off, definitely gonna squash merge this. That commit log is long and not that helpful... no problems there, I do the same, but in this case i think its appropriate to leave out all the iterations and merely accept the final diff.

File: README.md [hunk 1]
```diff
@@ -32,11 +32,26 @@ dotnet restore
...
+```bash 
+dotnet run
+```
+
+from the "DiscordBot" directory and the bot should come online (assuming you've completed the below steps). 
+
+## Discord Bot App Token and Server Integration
+You can find this information in more detail on the DiscordAPI and discord.net docs, but a summary: 
+
+1. To get a token: Go to https://discordapp.com/developers/applications/ --> "Create a new application" --> "Bot" on the sidebar on the left --> "Add Bot" --> click "Reveal Token" to get/see your token. This token should go into the corresponding field in config.json.
```
Comment (joefroh) [inline README.md]: can you refactor this line to be a bit more read able using markdown (like a bullet list or a numbered/lettered list) per major instruction

e.g.

1. To get a token:
      a) Go to https:// .....

**Comment by MRbiomedical** [comment]
Aight, I edited the stuff on my master and I believe this PR was updated accordingly (it doesn't show the updated version anywhere but you can see the three new commits. If you want I can close this PR and open a new one) 

Also yes please squash merge, my commit messages are useless

**Pull request merged**